### PR TITLE
Added README & Improved TinyMCE Documentation

### PR DESCRIPTION
**Pull request by schwindy**
## Overview
I added a `README.md` to the project and added a comment to `config/nova-tinymce-field.php` that links to the TinyMCE configuration documentation.

I also added the publish command to the Installation section of the README
```
php artisan vendor:publish --provider="Bythepixel\\NovaTinymceField\\FieldServiceProvider"
```

I also improved the default TinyMCE configuration to better persist non-breaking Microsoft Word formatting when copy-pasting from Word into the WYSIWIG 🐒

### CODE REVIEW

File: config/nova-tinymce-field.php [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+ *
+ * For more details, see: https://www.tiny.cloud/docs/configure/
+ */
 return [
     'options' => [
         'init' => [
-            'menubar' => false,
             'branding' => false,
             'image_caption' => true,
+            'menubar' => false,
```
Comment (rreynier) [inline config/nova-tinymce-field.php:15]: Your OCD is high level!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# nova-tinymce-field
+This package is a Nova WYSIWIG Field that uses TinyMCE, which has an extensive set of configuration/plugins.
+
+### Installation
+- `composer require bythepixel/nova-tinymce-field`
```
Comment (rreynier) [inline README.md:5]: Add the "publish" command also?

Comment (schwindy) [reply README.md:5]: @rreynier Yup, added it. Looks like this package doesn't have a "tag" so we have to publish like this, correct?
```
php artisan vendor:publish --provider="Bythepixel\\NovaTinymceField\\FieldServiceProvider"
```

Comment (rreynier) [reply README.md:5]: Correct.

**Review by rreynier — APPROVED** [review]
:+1:

**Pull request merged**